README.rst

```diff
@@ -70,6 +70,14 @@ Commercial Support
 ~~~~~~~~~~~~~~~~~~
 Wagtail is sponsored by `Torchbox <https://torchbox.com/>`_. If you need help implementing or hosting Wagtail, please contact us: hello@torchbox.com.
 
+Sponsors
+~~~~~~~~
+Wagtail is also sponsored by `BrowserStack <https://www.browserstack.com/>`_, who provide the project with free access to their live web-based browser testing tool, and automated Selenium cloud testing.
```

Comment:
'Sponsors' feels a bit strong. I suggest

## Credits (or Thanks)

We thank BrowserStack, who provide...